components/src/web-components/display/aggregate-component.tsx

```diff
@@ -46,8 +46,22 @@ export class AggregateComponent extends PreactLitAdapterWithGridJsStyles {
     @property({ type: Object })
     size: { width?: string; height?: string } | undefined = undefined;
 
+    /**
+     * The headline of the component. (optional)
```

Comment:
I tried to set `undefined` in the React example, I didn't manage to disable the heading with it.
```suggestion
     * The headline of the component. Set to an empty string to hide the headline.
```